iroh-net/src/discovery/local_swarm_discovery.rs

```diff
@@ -296,8 +296,16 @@ mod tests {
     use testresult::TestResult;
 
     #[tokio::test]
-    #[ignore = "flaky"]
+    // #[ignore = "flaky"]
     async fn test_local_swarm_discovery() -> TestResult {
+        let _guard = iroh_test::logging::setup();
+        for _i in 0..10 {
+            local_swarm_discovery().await?;
+        }
+        Ok(())
```

Comment:
should i keep this loop to catch any future flaky-ness sooner?

Reply:
Hm. It seems weird and somewhat arbitrary to me.
In the worst case this is going to be a test that runs nearly 100 seconds as part of the normal test suite in iroh-net. This could be annoying when developing.

Reply:
Hm. It seems weird and somewhat arbitrary to me.
In the worst case this is going to be a test that runs nearly 100 seconds as part of the normal test suite in iroh-net. This could be annoying when developing.

Reply:
extremely good point